### ISSUE TITLE
not sure what this error could be...

### ISSUE THREAD

**Issue by jAyATJaY** [volotat/SD-CN-Animation#17]
i updated the script to the latest version and now I cant seem to avoid this error. Anybody know what it means?

compute_flow.py: error: unrecognized arguments: Diffusion\stable-diffusion-webui\scripts\SD-CN-Animation-main\1.mp4 Diffusion\stable-diffusion-webui\scripts\SD-CN-Animation-main\flow.h5

i'm guessing the path structure is incorrect on my part. not sure but any help would be appreciated and please accept my humblest apology if this error was posted and solved somewhere else.

**Comment by volotat** [comment]
Please show the exact thing you run in the console.

**Comment by jAyATJaY** [comment]
python3 compute_flow.py -i {D:\Stable Diffusion\stable-diffusion-webui\scripts\SD-CN-Animation-main\1.mp4} -o {D:\Stable Diffusion\stable-diffusion-webui\scripts\SD-CN-Animation-main\flow.h5} -v -W 512 -H 512
this is what i ran. everything look ok or no?

**Comment by alexfredo** [comment]
Try :  py compute_flow.py -i D:\Stable Diffusion\stable-diffusion-webui\scripts\1.mp4 -o D:\Stable Diffusion\stable-diffusion-webui\scripts\flow.h5 -v -W 512 -H 512

**Comment by jAyATJaY** [comment]
> Try : py compute_flow.py -i D:\Stable Diffusion\stable-diffusion-webui\scripts\1.mp4 -o D:\Stable Diffusion\stable-diffusion-webui\scripts\flow.h5 -v -W 512 -H 512

Thank you so much for your help, I'll give it I try.

**Comment by alexfredo** [comment]
I written the wrong path I have updated my first comment

**Comment by volotat** [comment]
@jAyATJaY You are not supposed to use brackets in the console, it is just for input indication. I should probably update instruction to avoid this confusion.

**Comment by jAyATJaY** [comment]
Now I have this, 
Traceback (most recent call last):
  File "D:\Stable Diffusion\stable-diffusion-webui\scripts\SD-CN-Animation-main\compute_flow.py", line 1, in <module>
    import cv2
ModuleNotFoundError: No module named 'cv2'

**Comment by alexfredo** [comment]
Do you have install dependencies ?  : pip install opencv-python opencv-contrib-python numpy tqdm h5py

**Comment by jAyATJaY** [comment]
yes it says that are requirements are satisfied.

**Comment by volotat** [comment]
You might have issue with multiple python instances installed on the system. No. Try the following, run:
python3 -m pip freeze
Check if there is opencv-python in the list. If there is not, that means that when you run 'pip' command it interacts with the different version of python that is set to 'python3' command. If so, you should install opencv-python again with the following command:
python3 -m pip install opencv-python

**Comment by jAyATJaY** [comment]
okay so this is what it came up with. i do see opencv python.
accelerate==0.18.0
certifi==2022.12.7
charset-normalizer==2.1.1
colorama==0.4.6
diffusers==0.14.0
filelock==3.9.0
h5py==3.8.0
huggingface-hub==0.13.4
idna==3.4
importlib-metadata==6.3.0
Jinja2==3.1.2
MarkupSafe==2.1.2
mpmath==1.2.1
mypy-extensions==1.0.0
networkx==3.0
numpy==1.24.1
opencv-contrib-python==4.7.0.72
opencv-python==4.7.0.68
packaging==23.0
Pillow==9.3.0
psutil==5.9.4
pyre-extensions==0.0.23
PyYAML==6.0
regex==2023.3.23
requests==2.28.1
scipy==1.10.1
sympy==1.11.1
tokenizers==0.13.3
torch==2.0.0+cu118
torchvision==0.15.1+cu118
tqdm==4.65.0
transformers==4.27.4
typing-inspect==0.8.0
typing_extensions==4.4.0
urllib3==1.26.13
xformers==0.0.19.dev516
zipp==3.15.0

**Comment by volotat** [comment]
Ok. That's weird. If you use ubuntu you may try to do the following:
```
sudo apt update
sudo apt install libopencv-dev python3-opencv
```

**Comment by CaptnSeraph** [comment]
You could create a virtual environment for this, that's what I do to maintain separation from all the different versions of stuff I'm using.

**Comment by jAyATJaY** [comment]
Yes, I'm on Windows. I'm new to all of this. I'll keep trying. I'm really excited about trying this script. It looks like one of the best ones out there and I think I've tried them all. I think.

**Comment by jAyATJaY** [comment]
> You might have issue with multiple python instances installed on the system. Try the following, run: `python3 -m pip freeze` Check if there is opencv-python in the list. If there is not, that means that when you run 'pip' command it interacts with the different version of python that is set to 'python3' command. If so, you should install opencv-python again with the following command: `python3 -m pip install opencv-python`

you were right, i did discover multiple instances of python installed. went through the windows app store which i wasn't aware of. i just uninstalled both and reinstalled just to be sure. now everything's working as it should.
